build/assets/style.csl

```diff
@@ -50,26 +50,28 @@
   <bibliography hanging-indent="true" second-field-align="margin">
     <layout>
       <text variable="citation-number" suffix=". "/>
-      <group>
-        <text variable="title" font-weight="bold"/>
-      </group>
-      <group display="block">
-        <text macro="author"/>
-      </group>
       <group delimiter=" ">
-        <text macro="venue" font-style="italic"/>
-        <date variable="issued" prefix="(" suffix=")">
-          <date-part name="year"/>
-          <date-part name="month" form="numeric-leading-zeros" prefix="-"/>
-          <date-part name="day" form="numeric-leading-zeros" prefix="-"/>
-        </date>
-        <text variable="URL"/>
-      </group>
-      <group display="block" delimiter=" · ">
-        <text variable="DOI" text-case="lowercase" prefix="DOI: "/>
-        <text variable="PMID" prefix="PMID: "/>
-        <text variable="PMCID" prefix="PMCID: "/>
-        <text variable="ISBN" prefix="ISBN: "/>
+        <group>
+          <text variable="title" font-weight="bold"/>
+        </group>
+        <group display="block">
```

Comment:
> citeproc-js isn't happy with those two blocks in there

@adam3smith I'm guessing this comment is in reference to this line and the other one below:

```xml
        <group display="block" delimiter=" · ">
```

By not happy, do you mean that this style fails entirely with citeproc-js? Do you think the issue is with our style or citeproc-js? I.e. is this use of block valid according the spec?